src/app/models/video-item.model.ts

```diff
@@ -0,0 +1,44 @@
+export interface IVideoItem {
+  kind: string
+  etag: string
+  id: string
+  snippet: IVideoSnippet
+  statistics: IVideoStatistics
+}
+
+interface IVideoSnippet {
+  publishedAt: string
+  channelId: string
+  title: string
+  description: string
+  thumbnails: {
+    default: IVideoThumbnail
+    medium: IVideoThumbnail
+    high: IVideoThumbnail
+    standard: IVideoThumbnail
+    maxres: IVideoThumbnail
+  }
```

Comment:
I think it's good to avoid usage of nested structures.
Let's suppose the developer who reads this code is not interested in `thumbnails` property at all, and just wants to understand the overall structure of `IVideoSnippet`. If you separate `thumbnails` internals to it's own interface, you'll save the developer a few sec of time and a few bytes of brain RAM 🙂